astroplan/tests/test_observer.py

```diff
@@ -1297,3 +1297,30 @@ def test_moon_rise_set():
             datetime.timedelta(minutes=threshold_minutes))
     assert (abs(pyephem_prev_set - astroplan_prev_set.datetime) <
             datetime.timedelta(minutes=threshold_minutes))
+
+usno_sunsets = [
+    Time('2019-01-01 00:31'),
+    Time('2019-02-01 00:58'),
+    Time('2019-03-01 01:21'),
+    Time('2019-04-01 01:43'),
+    Time('2019-05-01 02:03'),
+    Time('2019-06-01 02:24'),
+]
+
+@pytest.mark.parametrize('usno_sunset', usno_sunsets)
+def test_sun_set_vs_usno_almanac(usno_sunset):
```

Comment:
i would specify that this is the MMTO almanac. i don't know if our code is actually derived from any USNO code. however, the parameters used in the code are those published by the USNO in their Astronomical Almanac (currently offline: http://asa.usno.navy.mil/) and the times have been verified in the past against the USNO online calculator.